src/figuresClasses.ts

```diff
@@ -1,19 +1,80 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape = 'triangle';
 
-}
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Side length should be greater than 0');
+    }
+
+    const arrayOfTriangle: number[] = [a, b, c]
+      .sort((x: number, y: number) => y - x);
 
-export class Circle {
+    if (arrayOfTriangle[0] === arrayOfTriangle[1] + arrayOfTriangle[2]) {
+      throw new Error('A triangle can\'t be created from these 3 sides');
+    }
+  }
 
+  getArea(): number {
+    const p: number = (this.a + this.b + this.c) / 2; // half of perimeter
+    const area: number = (p
+      * (p - this.a)
+      * (p - this.b)
+      * (p - this.c)) ** (1 / 2); // Heron's formula
+
+    return Number(area.toFixed(2));
```

Comment:
```suggestion
    return Number(area);
```
You can use `toFixed` only once at the end in your `getInfo` function